File: FITCC.UI/admin_fitcc/lib/models/kriterij.dart
import 'package:json_annotation/json_annotation.dart';

part 'kriterij.g.dart';

@JsonSerializable(explicitToJson: true)
class Kriterij {
  int kriterijId;
   String naziv;
   String vrijednost;
   int kategorijaId;

  Kriterij(
    this.kriterijId,
    this.naziv,
    this.vrijednost,
    this.kategorijaId);

  factory Kriterij.fromJson(Map<String, dynamic> json) => _$KriterijFromJson(json);
  Map<String, dynamic> toJson() => _$KriterijToJson(this);
}

File: FITCCRS2App.Models/Models/Kriterij.cs
namespace FITCCRS2App.Models.Models
{
    public class Kriterij
	{
        public int KriterijId { get; set; }
        public string? Naziv { get; set; }
        public string? Vrijednost { get; set; }

        public int? KategorijaId { get; set; }
        public virtual Kategorija? Kategorija { get; set; }
    }
}

